maven-core/src/test/java/org/apache/maven/project/ProjectBuilderTest.java

```diff
@@ -347,4 +349,32 @@ public void testBuildFromModelSourceResolvesBasedir() throws Exception {
         }
         assertEquals(0, errors);
     }
+
+    @Test
+    public void testLocationTrackingResolution() throws Exception {
+        File pom = getProject("MNG-7654");
```

Comment:
Should be `MNG-7648` - 7654 it is not issue for this PR.

Reply:
OMG how have I switched that :facepalm:

Reply:
Fixed (such a stupid mistake).